.github/workflows/ci.yml

```diff
@@ -32,4 +34,4 @@ jobs:
         tox --parallel auto --notest
     - name: Test with tox
       run: |
-        tox --parallel 0
+        tox --parallel 0 -v
```

Comment:
@Kriechi Do you mean to keep this, or was it for your development?

Reply:
dev work - I remove it.